docs/quickstart_using.md

```diff
@@ -1,6 +1,6 @@
 # Empire :: Using
 
-Going forward from the [installing](./installing.md) guide, the first thing we'll need to do is tell the empire client where it can find the empire API. The *bootstrap* command should have printed that out for you, so make sure you've set it in your environment like so:
+Going forward from the [installing](http://empire.readthedocs.org/en/latest/quickstart_installing/) guide, the first thing we'll need to do is tell the empire client where it can find the empire API. The *bootstrap* command should have printed that out for you, so make sure you've set it in your environment like so:
```

Comment:
Is there a better way to do relative links with readthedocs? Sucks that the github style doesn't work.